ruby/lib/ThreatExchange.rb

```diff
@@ -17,27 +17,16 @@ def initialize(access_token=nil)
       @baseurl = 'https://graph.facebook.com'
     end
 
-    def malware_analyses(filter={})
-      filter[:access_token] = @access_token
-      begin
-        response = RestClient.get "#{@baseurl}/malware_analyses",
-        { params: filter  }
-        result = JSON.parse(response)
-      rescue => e
-        puts e.inspect
-      end
+    def malware_analyses(params={})
+      params = params.merge(access_token: @access_token)
```

Comment:
also toyed w/the idea of moving this line from each method into the HTTP methods -- thoughts?